### PR TITLE
Refactor api access

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @ajacksified and I guess @xilvar

### CODE REVIEW

File: src/views/components/TopNav.jsx [hunk 1]
```diff
@@ -184,7 +184,7 @@ class TopNav extends React.Component {
         sr: state.subredditId
       });
 
-      var options = props.api.buildOptions(props.token, props.userAgent);
+      var options = this.props.api.buildOptions(this.props.apiOptions);
```
Comment (ajacksified) [inline src/views/components/TopNav.jsx]: Is `this` necessary?

Comment (ajacksified) [inline src/views/components/TopNav.jsx]: Nope, I'll fix :+1:

**Comment by ajacksified** [comment]
:fish:

**Comment by ajacksified** [comment]
(Also need to point to latest snoode; don't merge before https://github.com/reddit/snoode/pull/18)

**Comment by xilvar** [comment]
:fish: I suppose

File: src/oauth.es6.js [hunk 2]
```diff
@@ -8,7 +8,6 @@ var oauthRoutes = function(app) {
   var router = app.router;
 
   var cookieOptions = {
-    signed: true,
```
Comment (JordanMilne) [inline src/oauth.es6.js]: were signatures busting things?

Comment (ajacksified) [inline src/oauth.es6.js]: I think so.

Comment (ajacksified) [inline src/oauth.es6.js]: But I'll verify.

**Pull request merged**